README.md

```diff
@@ -1 +1,177 @@
-# https://www.kaggle.com/c/h-and-m-personalized-fashion-recommendations
\ No newline at end of file
+# **laplace**, a Graph Neural Network based Recommendation Engine
+
+##### Table of Contents  
+- [Overview](#Overview)  
+- [Get Started](#Get_Started) 
+  - [Installation & Data](#Installation) 
+  - [Main Pipeline](#Main_Pipeline) 
+- [Advanced Usage](#Advanced_Usage)  
+    - [Hyperparameter tuning](#Hyperparameter_tuning)     
+- [Further todo](#Further_todo)  
+- [Recommended resources](#Recommended_resources)  
+
+<br>
+
+
+
+<a name="Overview"></a>
+
+## Overview
+
+**laplace** is an end-to-end ML framework to train and predict on neurally-enhanced graphs for recommendation. 
+
+The pipeline is designed for self-supervised edge prediction on heterogenous graphs. 
+
+<br>
+
+![Pipeline Overview](/docs/assets/pipeline_overview.png)
+
+### Features
+
+- Two step, **Multi-model** composition:
+  1. Candidate Selection
```

Comment:
I think the usual terms are "Matching" & "Ranking" https://www.alibabacloud.com/blog/recommendation-system-matching-algorithms-and-architecture_596645 https://www.alibabacloud.com/blog/recommender-system-ranking-algorithms-and-training-architectures_596643

Reply:
Jure Leskovec used candidate selection throughout the lectures. At least for me candidate selection seems to describe what we are doing in the first step much better than matching.

Reply:
how about the second part? "Ranking"?